gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -237,7 +293,7 @@ static String selector(LicenseField... fields) {
       EQ,
 
       /**
-       * Defines an inequality filter.
+       * Defines a not-equals filter.
```

Comment:
Nit, with this change I think chaining the EQ javadoc to  "Defines an equals filter" would sound more consistent but fine as is.